csrc/apis/python/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@
 cmake_minimum_required(VERSION 3.14)
 project(mmdeploy_python)
 
+if ("cuda" IN_LIST MMDEPLOY_TARGET_DEVICES)
```

Comment:
Why on earth does the cmake list file for a language binding has to care about which devices are we building against?

We need to do better in the next release, or we will have a list of *if-device-enabled* in the future.